docs/orchestration/faq/upgrading_1.0.md

```diff
@@ -0,0 +1,165 @@
+- By default, for script-based storage such as GitHub, a flow will only be re-registered if it is *structurally different* than the existing version. This means that small edits to the source of tasks won't require re-registration, and the CLI will automatically detect this to avoid needlessly bumping the version. This can be disabled by passing in `--force`. If you pickle (default behavior) like Local storage, you do need to register for changes to take effect because you need the file in storage to change.
+- A `--watch` flag enables watching for changes in a directory, path, or module and re-registering flows if a change is detected. This can be used during development to automatically re-register your flows on save (as needed), or as part of a deployment for users who want to watch and auto-update flows from a specific directory.
+
+These features simplify auto-registering flows from within CI. For example, with GitHub actions you might add the following step to your CI workflow to auto-register all flows in a `flows/` directory on merge.
+
+```yaml
+- name: Register Flows
+  if: github.ref == 'refs/heads/master'
+  run: prefect register flows --project testing --path flows
+```
+
+The changes in flow registration require Prefect Server 2021.09.02 or later. Prefect Server will need to be upgraded before flows can be registered from this version.
+
+### Running flows
+
+The `prefect run` replaces the `prefect run flow` command to run a flow from the CLI.
+
+`prefect run` can run flows locally without the backend (Prefect Server or Prefect Cloud), or with the backend by submitting to an agent. It takes many options for lookup including a Python import name, a file path, the flow ID, the flow group ID, flow name, or project name. The flow run state change and log display has been entirely rewritten to be nice looking. 
+
+`prefect run` supports the following options:
+
+| Option | Description |
+| --- | --- |
+| -i, --id           | The UUID of a flow or flow group to run. If a flow group id is given, the latest flow id will be used for the run. |
+| --project          | The name of the Prefect project containing the flow to run. |
+| -p, --path         | The path to a file containing the flow to run. |
+| -m, --module       | The python module name containing the flow to run. |
+| -n, --name         | The name of a flow to run from the specified file/module/project. If the source contains multiple flows, this must be provided. |
+| --label            | A label to add to the flow run. May be passed multiple times to specify multiple labels. If not passed, the labels from the flow group will be used. |
+| --run-name         | A name to assign to the flow run. |
+| --con              | A key, value pair (key=value) specifying a flow con variable. The value will be interpreted as JSON. May be passed multiple times to specify multiple con values. Nested values may be set by passing a dict. |
```

Comment:
Can we link to what flow con variable is?

Reply:
This should be `--context` I think